### DISCUSSION TITLE
Enable addon on Multistre.am

### DISCUSSION THREAD

**Discussion by n-jones0** [night/betterttv#6911] [Ideas]
[Multistre.am](https://multistre.am/) is (one of?) the best ways to watch multiple streams at once, such as when people are doing co-op games. However, BetterTTV does not activate on the multistre.am website, so you lose the ability to see emotes. It would be great if it was made to activate on it.

BetterTTV Version: 7.6.5
Edge Version: 125.0.2535.85 (Official build) (64-bit)

**Comment by night** [comment]
The extension works on any available native Twitch chat embeds. It appears `Multistre.am` does embed the native Twitch chat, and the extension does load and render emotes when I test it.
